Add job to run all TinkerPop tests on schedule

Currently we don't run all TinkerPop tests against all storage backends because it takes very long time to execute all those tests. We should make a scheduled job to run all those tests.
Comment: https://github.com/JanusGraph/janusgraph/pull/1601#issuecomment-500404178
We could use the approach like this #1601

I would like to make a notice also that some tests are executed with errors when tinkerpop tests are not disabled. Can be seen here: https://travis-ci.org/porunov/janusgraph/builds/552172912
```
AstyanaxSSLStoreTest>KeyColumnValueStoreTest.setUp:60->AbstractCassandraStoreTest.openStorageManager:39->AbstractCassandraStoreTest.openStorageManager:165->AstyanaxStoreTest.openStorageManager:45 » TemporaryBackend
```
```
[ERROR] Errors: 
[ERROR]   CQLGraphTest>CassandraGraphTest.startCassandra:44 » IllegalArgument Path /home...
```

I didn't research what causes it. Maybe I just configured travis jobs with wrong parameters. But we should check it

@FlorianHockmann I used the command `mvn clean install -Dtest.skip.tp=false -Pdocker,solr7` after 13 hours and 27 minutes it was terminated because of failures.
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.janusgraph.diskstorage.es.BerkeleyElasticsearchTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.396 s <<< FAILURE! - in org.janusgraph.diskstorage.es.BerkeleyElasticsearchTest
[ERROR] org.janusgraph.diskstorage.es.BerkeleyElasticsearchTest  Time elapsed: 10.396 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at org.janusgraph.diskstorage.es.BerkeleyElasticsearchTest.<clinit>(BerkeleyElasticsearchTest.java:40)

[INFO] Running org.janusgraph.diskstorage.es.ElasticsearchConfigTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.363 s <<< FAILURE! - in org.janusgraph.diskstorage.es.ElasticsearchConfigTest
[ERROR] org.janusgraph.diskstorage.es.ElasticsearchConfigTest  Time elapsed: 10.363 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at org.janusgraph.diskstorage.es.ElasticsearchConfigTest.<clinit>(ElasticsearchConfigTest.java:76)

[INFO] Running org.janusgraph.diskstorage.es.ElasticsearchIndexTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.331 s <<< FAILURE! - in org.janusgraph.diskstorage.es.ElasticsearchIndexTest
[ERROR] org.janusgraph.diskstorage.es.ElasticsearchIndexTest  Time elapsed: 10.33 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at org.janusgraph.diskstorage.es.ElasticsearchIndexTest.<clinit>(ElasticsearchIndexTest.java:74)

[INFO] Running org.janusgraph.diskstorage.es.ElasticsearchMultiTypeIndexTest
[WARNING] Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.001 s - in org.janusgraph.diskstorage.es.ElasticsearchMultiTypeIndexTest
[INFO] Running org.janusgraph.diskstorage.es.ThriftElasticsearchTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.451 s <<< FAILURE! - in org.janusgraph.diskstorage.es.ThriftElasticsearchTest
[ERROR] org.janusgraph.diskstorage.es.ThriftElasticsearchTest  Time elapsed: 10.451 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at org.janusgraph.diskstorage.es.ThriftElasticsearchTest.<clinit>(ThriftElasticsearchTest.java:30)

[INFO] Running org.janusgraph.diskstorage.es.rest.RestClientSetupTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[INFO] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.487 s - in org.janusgraph.diskstorage.es.rest.RestClientSetupTest
[INFO] Running org.janusgraph.diskstorage.es.rest.util.BasicAuthHttpClientConfigCallbackTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.058 s - in org.janusgraph.diskstorage.es.rest.util.BasicAuthHttpClientConfigCallbackTest
[INFO] Running org.janusgraph.diskstorage.es.rest.util.SSLConfigurationCallbackTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.189 s - in org.janusgraph.diskstorage.es.rest.util.SSLConfigurationCallbackTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   BerkeleyElasticsearchTest » ExceptionInInitializer
[ERROR]   ElasticsearchConfigTest » ExceptionInInitializer
[ERROR]   ElasticsearchIndexTest » ExceptionInInitializer
[ERROR]   ThriftElasticsearchTest » ExceptionInInitializer
[INFO] 
[ERROR] Tests run: 30, Failures: 0, Errors: 4, Skipped: 1
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] JanusGraph: Distributed Graph Database 0.4.0-SNAPSHOT SUCCESS [  2.081 s]
[INFO] JanusGraph-Core: Core Library for JanusGraph ....... SUCCESS [ 19.229 s]
[INFO] JanusGraph-Test: Test Suite for JanusGraph ......... SUCCESS [41:43 min]
[INFO] JanusGraph CodePipelines CI: Distributed release testing. SUCCESS [  7.150 s]
[INFO] janusgraph-server .................................. SUCCESS [ 22.451 s]
[INFO] JanusGraph-BerkeleyJE: Distributed Graph Database .. SUCCESS [  02:04 h]
[INFO] JanusGraph-Cassandra: Distributed Graph Database ... SUCCESS [  05:59 h]
[INFO] JanusGraph-CQL: Distributed Graph Database ......... SUCCESS [  04:39 h]
[INFO] JanusGraph-HBase: Parent Module .................... SUCCESS [  0.096 s]
[INFO] JanusGraph-HBase: Version-independent Core ......... SUCCESS [  0.736 s]
[INFO] JanusGraph-ElasticSearch: Distributed Indexing Support FAILURE [ 55.150 s]
[INFO] JanusGraph-HBase: HBase Server Used by Maven Testing SKIPPED
[INFO] JanusGraph-HBase: 1.x and 2.x Compatibility Shim ... SKIPPED
[INFO] JanusGraph-HBase: Universal binary ................. SKIPPED
[INFO] JanusGraph-Hadoop: Parent Module ................... SKIPPED
[INFO] JanusGraph-Hadoop: Version-independent Core ........ SKIPPED
[INFO] JanusGraph-Hadoop: 2.x Compatibility Shim .......... SKIPPED
[INFO] JanusGraph-Hadoop: Universal binary ................ SKIPPED
[INFO] JanusGraph-Bigtable: Google Cloud Bigtable driver .. SKIPPED
[INFO] JanusGraph-Lucene: Indexing Support ................ SKIPPED
[INFO] JanusGraph-Solr: Distributed Indexing Support ...... SKIPPED
[INFO] JanusGraph-All: Complete JanusGraph Distribution ... SKIPPED
[INFO] JanusGraph-Dist: Tar and Zip Archives .............. SKIPPED
[INFO] JanusGraph-Dist: Archive with Hadoop 2 ............. SKIPPED
[INFO] JanusGraph-Doc: AsciiDoc Manual for JanusGraph ..... SKIPPED
[INFO] JanusGraph-Examples: Examples for JanusGraph ....... SKIPPED
[INFO] Example-Common: Common Graph Code for Examples ..... SKIPPED
[INFO] Example-BerkeleyJE: BerkeleyJE Storage, Lucene Index SKIPPED
[INFO] Example-Cassandra: Cassandra Thrift Storage, Elasticsearch Index SKIPPED
[INFO] Example-Cql: Cassandra CQL Storage, Elasticsearch Index SKIPPED
[INFO] Example-HBase: HBase Storage, Solr Index ........... SKIPPED
[INFO] Example-RemoteGraph: Example with RemoteGraph ...... SKIPPED
[INFO] Example-TinkerGraph: Example with TinkerGraph 0.4.0-SNAPSHOT SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13:27 h
[INFO] Finished at: 2019-07-01T13:38:56+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project janusgraph-es: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/alex/github/janusgraph/janusgraph/janusgraph-es/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :janusgraph-es
```

As you can see only 10 of 33 testing jobs was finished in 13:27 hours.

@FlorianHockmann I think I need to reinstall and configure docker somehow. I didn't work with docker before. I tried `docker run hello-world` it failed because the docker service is stopped. Then I tried to enable the docker and run `docker run hello-world` again and it failed due to permission denial. Then I tried to run `docker run hello-world` from the `root` and it worked. But `mvn clean install -rf :janusgraph-es -Dtest.skip.tp=false -Pdocker,solr7` doesn't work.
`janusgraph-es/target/test.log` shows:
```
2019-07-02 17:55:43.939 [main] ERROR dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy: ping failed with configuration Environment variables, system properties and defaults. Resolved: 
    dockerHost=unix:///var/run/docker.sock
    apiVersion='{UNKNOWN_VERSION}'
    registryUrl='https://index.docker.io/v1/'
    registryUsername='alex'
    registryPassword='null'
    registryEmail='null'
    dockerConfig='DefaultDockerClientConfig[dockerHost=unix:///var/run/docker.sock,registryUsername=alex,registryPassword=<null>,registryEmail=<null>,registryUrl=https://index.docker.io/v1/,dockerConfigPath=/home/alex/.docker,sslConfig=<null>,apiVersion={UNKNOWN_VERSION},dockerConfig=<null>]'
 due to org.rnorth.ducttape.TimeoutException: Timeout waiting for result with exception
org.rnorth.ducttape.TimeoutException: Timeout waiting for result with exception
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:51)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.ping(DockerClientProviderStrategy.java:190)
	at org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy.test(EnvironmentAndSystemPropertyClientProviderStrategy.java:42)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$2(DockerClientProviderStrategy.java:113)
	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267)
	at java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(StreamSpliterators.java:302)
	at java.util.stream.Streams$ConcatSpliterator.tryAdvance(Streams.java:731)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findAny(ReferencePipeline.java:469)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:148)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:105)
	at org.testcontainers.containers.GenericContainer.<init>(GenericContainer.java:142)
	at org.testcontainers.elasticsearch.ElasticsearchContainer.<init>(ElasticsearchContainer.java:48)
	at org.janusgraph.diskstorage.es.JanusGraphElasticsearchContainer.<init>(JanusGraphElasticsearchContainer.java:54)
	at org.janusgraph.diskstorage.es.JanusGraphElasticsearchContainer.<init>(JanusGraphElasticsearchContainer.java:50)
	at org.janusgraph.diskstorage.es.BerkeleyElasticsearchTest.<clinit>(BerkeleyElasticsearchTest.java:40)
	at sun.misc.Unsafe.ensureClassInitialized(Native Method)
	at sun.reflect.UnsafeFieldAccessorFactory.newFieldAccessor(UnsafeFieldAccessorFactory.java:43)
	at sun.reflect.ReflectionFactory.newFieldAccessor(ReflectionFactory.java:156)
	at java.lang.reflect.Field.acquireFieldAccessor(Field.java:1088)
	at java.lang.reflect.Field.getFieldAccessor(Field.java:1069)
	at java.lang.reflect.Field.get(Field.java:393)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.getContainerInstance(TestcontainersExtension.java:109)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$findSharedContainers$5(TestcontainersExtension.java:71)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.beforeAll(TestcontainersExtension.java:39)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.lambda$invokeBeforeAllCallbacks$8(ClassTestDescriptor.java:361)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.invokeBeforeAllCallbacks(ClassTestDescriptor.java:361)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.before(ClassTestDescriptor.java:197)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.before(ClassTestDescriptor.java:77)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:229)
	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:197)
	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:211)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:191)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.io.IOException: com.sun.jna.LastErrorException: [13]
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocket.<init>(UnixDomainSocket.java:62)
	at org.testcontainers.dockerclient.transport.okhttp.UnixSocketFactory$1.<init>(UnixSocketFactory.java:27)
	at org.testcontainers.dockerclient.transport.okhttp.UnixSocketFactory.createSocket(UnixSocketFactory.java:27)
	at org.testcontainers.shaded.okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)
	at org.testcontainers.shaded.okhttp3.internal.connection.RealConnection.connect(RealConnection.java:166)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at org.testcontainers.shaded.okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:254)
	at org.testcontainers.shaded.okhttp3.RealCall.execute(RealCall.java:92)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.execute(OkHttpInvocationBuilder.java:259)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.execute(OkHttpInvocationBuilder.java:254)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.get(OkHttpInvocationBuilder.java:220)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpDockerCmdExecFactory$1.execute(OkHttpDockerCmdExecFactory.java:124)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpDockerCmdExecFactory$1.execute(OkHttpDockerCmdExecFactory.java:117)
	at com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21)
	at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$null$4(DockerClientProviderStrategy.java:193)
	at org.rnorth.ducttape.ratelimits.RateLimiter.getWhenReady(RateLimiter.java:51)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$ping$5(DockerClientProviderStrategy.java:191)
	at org.rnorth.ducttape.unreliables.Unreliables.lambda$retryUntilSuccess$0(Unreliables.java:41)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.sun.jna.LastErrorException: [13]
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocketLibrary.connect(Native Method)
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocket.<init>(UnixDomainSocket.java:57)
	... 35 more
2019-07-02 17:56:03.970 [main] ERROR dockerclient.DockerClientProviderStrategy: Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
2019-07-02 17:56:03.971 [main] ERROR dockerclient.DockerClientProviderStrategy:     EnvironmentAndSystemPropertyClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed)
2019-07-02 17:56:03.971 [main] ERROR dockerclient.DockerClientProviderStrategy:     UnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause LastErrorException ([13] Àw
)
2019-07-02 17:56:03.971 [main] ERROR dockerclient.DockerClientProviderStrategy:     ProxiedUnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause TimeoutException (null)
2019-07-02 17:56:03.971 [main] ERROR dockerclient.DockerClientProviderStrategy: As no valid configuration was found, execution cannot continue
2019-07-02 17:56:15.106 [main] ERROR dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy: ping failed with configuration Environment variables, system properties and defaults. Resolved: 
    dockerHost=unix:///var/run/docker.sock
    apiVersion='{UNKNOWN_VERSION}'
    registryUrl='https://index.docker.io/v1/'
    registryUsername='alex'
    registryPassword='null'
    registryEmail='null'
    dockerConfig='DefaultDockerClientConfig[dockerHost=unix:///var/run/docker.sock,registryUsername=alex,registryPassword=<null>,registryEmail=<null>,registryUrl=https://index.docker.io/v1/,dockerConfigPath=/home/alex/.docker,sslConfig=<null>,apiVersion={UNKNOWN_VERSION},dockerConfig=<null>]'
 due to org.rnorth.ducttape.TimeoutException: Timeout waiting for result with exception
org.rnorth.ducttape.TimeoutException: Timeout waiting for result with exception
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:51)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.ping(DockerClientProviderStrategy.java:190)
	at org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy.test(EnvironmentAndSystemPropertyClientProviderStrategy.java:42)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$2(DockerClientProviderStrategy.java:113)
	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267)
	at java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(StreamSpliterators.java:302)
	at java.util.stream.Streams$ConcatSpliterator.tryAdvance(Streams.java:731)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findAny(ReferencePipeline.java:469)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:148)
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:105)
	at org.testcontainers.containers.GenericContainer.<init>(GenericContainer.java:142)
	at org.testcontainers.elasticsearch.ElasticsearchContainer.<init>(ElasticsearchContainer.java:48)
	at org.janusgraph.diskstorage.es.JanusGraphElasticsearchContainer.<init>(JanusGraphElasticsearchContainer.java:54)
	at org.janusgraph.diskstorage.es.JanusGraphElasticsearchContainer.<init>(JanusGraphElasticsearchContainer.java:50)
	at org.janusgraph.diskstorage.es.ElasticsearchConfigTest.<clinit>(ElasticsearchConfigTest.java:76)
	at sun.misc.Unsafe.ensureClassInitialized(Native Method)
	at sun.reflect.UnsafeFieldAccessorFactory.newFieldAccessor(UnsafeFieldAccessorFactory.java:43)
	at sun.reflect.ReflectionFactory.newFieldAccessor(ReflectionFactory.java:156)
	at java.lang.reflect.Field.acquireFieldAccessor(Field.java:1088)
	at java.lang.reflect.Field.getFieldAccessor(Field.java:1069)
	at java.lang.reflect.Field.get(Field.java:393)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.getContainerInstance(TestcontainersExtension.java:109)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.lambda$findSharedContainers$5(TestcontainersExtension.java:71)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at org.testcontainers.junit.jupiter.TestcontainersExtension.beforeAll(TestcontainersExtension.java:39)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.lambda$invokeBeforeAllCallbacks$8(ClassTestDescriptor.java:361)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.invokeBeforeAllCallbacks(ClassTestDescriptor.java:361)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.before(ClassTestDescriptor.java:197)
	at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.before(ClassTestDescriptor.java:77)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:229)
	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:197)
	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:211)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:191)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.io.IOException: com.sun.jna.LastErrorException: [13]
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocket.<init>(UnixDomainSocket.java:62)
	at org.testcontainers.dockerclient.transport.okhttp.UnixSocketFactory$1.<init>(UnixSocketFactory.java:27)
	at org.testcontainers.dockerclient.transport.okhttp.UnixSocketFactory.createSocket(UnixSocketFactory.java:27)
	at org.testcontainers.shaded.okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)
	at org.testcontainers.shaded.okhttp3.internal.connection.RealConnection.connect(RealConnection.java:166)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at org.testcontainers.shaded.okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at org.testcontainers.shaded.okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at org.testcontainers.shaded.okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at org.testcontainers.shaded.okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:254)
	at org.testcontainers.shaded.okhttp3.RealCall.execute(RealCall.java:92)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.execute(OkHttpInvocationBuilder.java:259)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.execute(OkHttpInvocationBuilder.java:254)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpInvocationBuilder.get(OkHttpInvocationBuilder.java:220)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpDockerCmdExecFactory$1.execute(OkHttpDockerCmdExecFactory.java:124)
	at org.testcontainers.dockerclient.transport.okhttp.OkHttpDockerCmdExecFactory$1.execute(OkHttpDockerCmdExecFactory.java:117)
	at com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21)
	at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$null$4(DockerClientProviderStrategy.java:193)
	at org.rnorth.ducttape.ratelimits.RateLimiter.getWhenReady(RateLimiter.java:51)
	at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$ping$5(DockerClientProviderStrategy.java:191)
	at org.rnorth.ducttape.unreliables.Unreliables.lambda$retryUntilSuccess$0(Unreliables.java:41)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.sun.jna.LastErrorException: [13]
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocketLibrary.connect(Native Method)
	at org.testcontainers.shaded.org.scalasbt.ipcsocket.UnixDomainSocket.<init>(UnixDomainSocket.java:57)
	... 35 more
```

I will try to reinstall it a little bit later and run those test again.

2 tests finished and failures again.
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.janusgraph.diskstorage.hbase.HBaseDistributedStoreManagerTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.087 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseDistributedStoreManagerTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseDistributedStoreManagerTest  Time elapsed: 1.087 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseDistributedStoreManagerTest.startHBase(HBaseDistributedStoreManagerTest.java:32)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseIDAuthorityTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.103 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseIDAuthorityTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseIDAuthorityTest  Time elapsed: 1.103 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseIDAuthorityTest.startHBase(HBaseIDAuthorityTest.java:30)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseLockStoreTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.095 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseLockStoreTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseLockStoreTest  Time elapsed: 1.095 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseLockStoreTest.startHBase(HBaseLockStoreTest.java:31)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseLogTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.106 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseLogTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseLogTest  Time elapsed: 1.106 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseLogTest.startHBase(HBaseLogTest.java:31)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseMultiWriteStoreTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.103 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseMultiWriteStoreTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseMultiWriteStoreTest  Time elapsed: 1.102 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseMultiWriteStoreTest.startHBase(HBaseMultiWriteStoreTest.java:30)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseStoreManagerConfigTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.09 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseStoreManagerConfigTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseStoreManagerConfigTest  Time elapsed: 1.09 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseStoreManagerConfigTest.startHBase(HBaseStoreManagerConfigTest.java:45)

[INFO] Running org.janusgraph.diskstorage.hbase.HBaseStoreManagerMutationTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 73.226 s - in org.janusgraph.diskstorage.hbase.HBaseStoreManagerMutationTest
[INFO] Running org.janusgraph.diskstorage.hbase.HBaseStoreTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.137 s <<< FAILURE! - in org.janusgraph.diskstorage.hbase.HBaseStoreTest
[ERROR] org.janusgraph.diskstorage.hbase.HBaseStoreTest  Time elapsed: 1.137 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.diskstorage.hbase.HBaseStoreTest.startHBase(HBaseStoreTest.java:36)

[INFO] Running org.janusgraph.graphdb.hbase.HBaseGraphTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.097 s <<< FAILURE! - in org.janusgraph.graphdb.hbase.HBaseGraphTest
[ERROR] org.janusgraph.graphdb.hbase.HBaseGraphTest  Time elapsed: 1.097 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.graphdb.hbase.HBaseGraphTest.startHBase(HBaseGraphTest.java:30)

[INFO] Running org.janusgraph.graphdb.hbase.HBaseOLAPTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.091 s <<< FAILURE! - in org.janusgraph.graphdb.hbase.HBaseOLAPTest
[ERROR] org.janusgraph.graphdb.hbase.HBaseOLAPTest  Time elapsed: 1.091 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.graphdb.hbase.HBaseOLAPTest.startHBase(HBaseOLAPTest.java:28)

[INFO] Running org.janusgraph.graphdb.hbase.HBaseOperationCountingTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.105 s <<< FAILURE! - in org.janusgraph.graphdb.hbase.HBaseOperationCountingTest
[ERROR] org.janusgraph.graphdb.hbase.HBaseOperationCountingTest  Time elapsed: 1.105 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.graphdb.hbase.HBaseOperationCountingTest.startHBase(HBaseOperationCountingTest.java:37)

[INFO] Running org.janusgraph.graphdb.hbase.HBasePartitionGraphTest
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/org/slf4j/slf4j-log4j12/1.7.12/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/alex/.m2/repository/ch/qos/logback/logback-classic/1.1.3/logback-classic-1.1.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.104 s <<< FAILURE! - in org.janusgraph.graphdb.hbase.HBasePartitionGraphTest
[ERROR] org.janusgraph.graphdb.hbase.HBasePartitionGraphTest  Time elapsed: 1.104 s  <<< ERROR!
java.lang.RuntimeException: Command "/home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin//hbase-daemon.sh --config /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/ start master" exited with status 1
	at org.janusgraph.graphdb.hbase.HBasePartitionGraphTest.startHBase(HBasePartitionGraphTest.java:37)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   HBaseDistributedStoreManagerTest.startHBase:32 » Runtime Command "/home/alex/D...
[ERROR]   HBaseIDAuthorityTest.startHBase:30 » Runtime Command "/home/alex/Documents/App...
[ERROR]   HBaseLockStoreTest.startHBase:31 » Runtime Command "/home/alex/Documents/Apps/...
[ERROR]   HBaseLogTest.startHBase:31 » Runtime Command "/home/alex/Documents/Apps/janusg...
[ERROR]   HBaseMultiWriteStoreTest.startHBase:30 » Runtime Command "/home/alex/Documents...
[ERROR]   HBaseStoreManagerConfigTest.startHBase:45 » Runtime Command "/home/alex/Docume...
[ERROR]   HBaseStoreTest.startHBase:36 » Runtime Command "/home/alex/Documents/Apps/janu...
[ERROR]   HBaseGraphTest.startHBase:30 » Runtime Command "/home/alex/Documents/Apps/janu...
[ERROR]   HBaseOLAPTest.startHBase:28 » Runtime Command "/home/alex/Documents/Apps/janus...
[ERROR]   HBaseOperationCountingTest.startHBase:37 » Runtime Command "/home/alex/Documen...
[ERROR]   HBasePartitionGraphTest.startHBase:37 » Runtime Command "/home/alex/Documents/...
[INFO] 
[ERROR] Tests run: 13, Failures: 0, Errors: 11, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] JanusGraph-ElasticSearch: Distributed Indexing Support SUCCESS [13:45 min]
[INFO] JanusGraph-HBase: HBase Server Used by Maven Testing SUCCESS [  4.628 s]
[INFO] JanusGraph-HBase: 1.x and 2.x Compatibility Shim ... FAILURE [01:35 min]
[INFO] JanusGraph-HBase: Universal binary ................. SKIPPED
[INFO] JanusGraph-Hadoop: Parent Module ................... SKIPPED
[INFO] JanusGraph-Hadoop: Version-independent Core ........ SKIPPED
[INFO] JanusGraph-Hadoop: 2.x Compatibility Shim .......... SKIPPED
[INFO] JanusGraph-Hadoop: Universal binary ................ SKIPPED
[INFO] JanusGraph-Bigtable: Google Cloud Bigtable driver .. SKIPPED
[INFO] JanusGraph-Lucene: Indexing Support ................ SKIPPED
[INFO] JanusGraph-Solr: Distributed Indexing Support ...... SKIPPED
[INFO] JanusGraph-All: Complete JanusGraph Distribution ... SKIPPED
[INFO] JanusGraph-Dist: Tar and Zip Archives .............. SKIPPED
[INFO] JanusGraph-Dist: Archive with Hadoop 2 ............. SKIPPED
[INFO] JanusGraph-Doc: AsciiDoc Manual for JanusGraph ..... SKIPPED
[INFO] JanusGraph-Examples: Examples for JanusGraph ....... SKIPPED
[INFO] Example-Common: Common Graph Code for Examples ..... SKIPPED
[INFO] Example-BerkeleyJE: BerkeleyJE Storage, Lucene Index SKIPPED
[INFO] Example-Cassandra: Cassandra Thrift Storage, Elasticsearch Index SKIPPED
[INFO] Example-Cql: Cassandra CQL Storage, Elasticsearch Index SKIPPED
[INFO] Example-HBase: HBase Storage, Solr Index ........... SKIPPED
[INFO] Example-RemoteGraph: Example with RemoteGraph ...... SKIPPED
[INFO] Example-TinkerGraph: Example with TinkerGraph ...... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15:25 min
[INFO] Finished at: 2019-07-02T20:56:19+03:00
[INFO] Final Memory: 104M/1866M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project janusgraph-hbase-10: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :janusgraph-hbase-10
```

the `test.log` is:
```
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
2019-07-02 21:02:09.055 [ReadOnlyZKClient-localhost:2181@0x06127a7e-SendThread(jblur:2181)] INFO  zookeeper.ClientCnxn: Opening socket connection to server jblur/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
2019-07-02 21:02:09.056 [ReadOnlyZKClient-localhost:2181@0x06127a7e-SendThread(jblur:2181)] WARN  zookeeper.ClientCnxn: Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
2019-07-02 21:02:09.204 [ReadOnlyZKClient-localhost:2181@0x45efc20d-SendThread(jblur:2181)] INFO  zookeeper.ClientCnxn: Opening socket connection to server jblur/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error)
2019-07-02 21:02:09.204 [ReadOnlyZKClient-localhost:2181@0x45efc20d-SendThread(jblur:2181)] WARN  zookeeper.ClientCnxn: Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
2019-07-02 21:02:09.305 [ReadOnlyZKClient-localhost:2181@0x45efc20d-SendThread(localhost:2181)] INFO  zookeeper.ClientCnxn: Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error)
2019-07-02 21:02:09.305 [ReadOnlyZKClient-localhost:2181@0x45efc20d] WARN  zookeeper.ReadOnlyZKClient: 0x45efc20d to localhost:2181 failed for get of /hbase/hbaseid, code = CONNECTIONLOSS, retries = 30, give up
2019-07-02 21:02:09.305 [main] WARN  client.ConnectionImplementation: Retrieve cluster id failed
java.util.concurrent.ExecutionException: org.apache.hadoop.hbase.shaded.org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1895)
	at org.apache.hadoop.hbase.client.ConnectionImplementation.retrieveClusterId(ConnectionImplementation.java:549)
	at org.apache.hadoop.hbase.client.ConnectionImplementation.<init>(ConnectionImplementation.java:287)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:220)
	at org.apache.hadoop.hbase.client.ConnectionFactory.createConnection(ConnectionFactory.java:115)
	at org.janusgraph.diskstorage.hbase.HBaseCompat1_0.createConnection(HBaseCompat1_0.java:43)
	at org.janusgraph.diskstorage.hbase.HBaseStoreManager.<init>(HBaseStoreManager.java:335)
	at org.janusgraph.diskstorage.hbase.HBaseStoreManagerMutationTest.testKCVMutationToPuts(HBaseStoreManagerMutationTest.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:628)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:117)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:184)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:180)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:127)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:135)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:229)
	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:197)
	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:211)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:191)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:142)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: org.apache.hadoop.hbase.shaded.org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /hbase/hbaseid
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.KeeperException.create(KeeperException.java:99)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.hadoop.hbase.zookeeper.ReadOnlyZKClient$ZKTask$1.exec(ReadOnlyZKClient.java:198)
	at org.apache.hadoop.hbase.zookeeper.ReadOnlyZKClient.run(ReadOnlyZKClient.java:342)
	at java.lang.Thread.run(Thread.java:748)
2019-07-02 21:02:09.306 [ReadOnlyZKClient-localhost:2181@0x45efc20d-SendThread(localhost:2181)] WARN  zookeeper.ClientCnxn: Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.hadoop.hbase.shaded.org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1141)
2019-07-02 21:02:10.225 [main] INFO  janusgraph.HBaseStorageSetup: HBase is not running
2019-07-02 21:02:10.226 [main] INFO  janusgraph.HBaseStorageSetup: Starting HBase
2019-07-02 21:02:10.227 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin/
2019-07-02 21:02:10.227 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/
2019-07-02 21:02:12.239 [main] INFO  janusgraph.HBaseStorageSetup: HBase is not running
2019-07-02 21:02:12.239 [main] INFO  janusgraph.HBaseStorageSetup: Starting HBase
2019-07-02 21:02:12.240 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin/
2019-07-02 21:02:12.241 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/
2019-07-02 21:02:14.126 [main] INFO  janusgraph.HBaseStorageSetup: HBase is not running
2019-07-02 21:02:14.127 [main] INFO  janusgraph.HBaseStorageSetup: Starting HBase
2019-07-02 21:02:14.128 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin/
2019-07-02 21:02:14.128 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/
2019-07-02 21:02:16.017 [main] INFO  janusgraph.HBaseStorageSetup: HBase is not running
2019-07-02 21:02:16.018 [main] INFO  janusgraph.HBaseStorageSetup: Starting HBase
2019-07-02 21:02:16.020 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin/
2019-07-02 21:02:16.020 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/
2019-07-02 21:02:17.965 [main] INFO  janusgraph.HBaseStorageSetup: HBase is not running
2019-07-02 21:02:17.966 [main] INFO  janusgraph.HBaseStorageSetup: Starting HBase
2019-07-02 21:02:17.968 [main] DEBUG janusgraph.HBaseStorageSetup: Built bin path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/bin/
2019-07-02 21:02:17.968 [main] DEBUG janusgraph.HBaseStorageSetup: Built conf path for HBase version 2.1.5: /home/alex/Documents/Apps/janusgraph/janusgraph-hbase-parent/janusgraph-hbase-10/../../janusgraph-hbase-parent/janusgraph-hbase-server/conf/
```

Looks like something is preventing from connecting to the port 2181 but no processes are listening this port on my machine.

--- a/.github/workflows/ci-release.yml
+++ b/.github/workflows/ci-release.yml
@@ -31,6 +31,8 @@ on:
     - 'requirements.txt'
     - 'docs.Dockerfile'
     - '*.md'
+  schedule:
+    - cron:  '0 0 * * 3'
 
 env:
   ES_JAVA_OPTS: "-Xms256m -Xmx512m"
@@ -78,7 +80,7 @@ jobs:
 
   tp-tests:
     runs-on: ubuntu-20.04
-    if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[tp-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[tp-tests]')"
+    if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[tp-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[tp-tests]') || github.event_name == 'schedule'"
     needs: build-all
     strategy:
       fail-fast: false

